docs/_sass/font-awesome/_animated.scss

```diff
@@ -1,20 +1,152 @@
-// Animated Icons
+// animating icons
 // --------------------------
 
+.#{$fa-css-prefix}-beat {
```

Comment:
surely we can update this to not include all of the raw scss, possible to pull it in via JS tags?

Reply:
( maybe worth trying in a nother pr )